packages/create-block-app/create-block-app.js

```diff
@@ -40,6 +40,7 @@ const templatePackageName = "block-template";
 
   await pacote.extract(templatePackageName, resolvedBlockPath, {
     registry: process.env.NPM_CONFIG_REGISTRY,
+    defaultTag: "beta",
```

Comment:
It’s simple to forget to undo this. How about leveraging `process.env.SOMETHING` instead?